src/client/pages/searchPage/Organisms/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,24 @@
+import { useDispatch } from 'react-redux';
+
+import styles from './SearchBar.module.css';
+
+import { SearchTypeSelector } from '../../Atoms/SearchTypeSelector/SearchTypeSelector.jsx';
+import { SearchForm } from '../../Molecules/SearchForm/SearchForm.jsx';
+import { searchRecipesByName } from '../../../../redux/reducers/search/actions.js';
```

Comment:
#path

Reply:
Answer in the same case have been already left.